Core/src/main/java/com/theprogrammingturkey/comz/listeners/SignListener.java

```diff
@@ -126,6 +126,12 @@ public void RightClickSign(PlayerInteractEvent event)
 			Sign sign = (Sign) event.getClickedBlock().getState();
 			Player player = event.getPlayer();
 
+			if (!event.getPlayer().isSneaking() && GameManager.INSTANCE.isPlayerInGame(player)) {
```

Comment:
What does sneaking have to do with anything here?

Reply:
don't know :p